### ISSUE TITLE
osutop

### ISSUE THREAD

**Issue by LiveEmily** [dain98/Minccino#10]
On the readme it says there is an -osutop command, even tho the bot doesn't react to it and it isn't documantated into the help command, pls fix

**Comment by dain98** [comment]
you're absolutely right, I'll update this issue as soon as I readd that command. Sorry I didn't see this for so long
